src/NEO/Transaction.cpp

```diff
@@ -0,0 +1,97 @@
+#include <ctype.h>
+
+#include "../uint256.h"
+#include "../UInt.hpp"
```

Comment:
for consistency, can we keep using `.h`? not `.hpp`

Reply:
Done for all files.